gpii/node_modules/eventLog/src/metrics.js

```diff
@@ -28,6 +28,10 @@ fluid.defaults("gpii.metrics", {
     contextAwareness: {
         platform: {
             checks: {
+                test: {
+                    contextValue: "{gpii.contexts.windows}",
```

Comment:
Not quite clear what went on here - you said that "The tests weren't expecting to be running under windows" but now you have written a rule which says, "If we are running under Windows, it must be a test", which doesn't seem quite right. What if we really are running under Windows? Suggests that this directive will break the system when we are running in real life. And if it doesn't, that suggests a further problem with how our contexts are registered :) Instead, there is a context named "gpii.contexts.test" which is the one that determines whether we are in a test.

Reply:
oops.

Well, it made the test pass.